ocw_data_parser/ocw_data_parser.py

```diff
@@ -198,6 +198,28 @@ def generate_master_json(self):
                             course_feature["ocw_feature_url"] = './resolveuid/' + page["uid"]
                             course_features[page["uid"]] = course_feature
         new_json["course_features"] = list(course_features.values())
+        courselist_features = []
+        courselist_features_raw = safe_get(self.jsons[0], "courselist_features")
```

Comment:
Do you have an example course with data for this value?

Reply:
I didn't want to bloat the library with an entire other example course, so I just edited the `1.json` file in one of the existing test courses to add the proper `courselist_features` entries.  We probably don't even need all the JSON we have in there, it could probably be more carefully curated.  It did the job back when this project had no tests and I had to add something though.

Reply:
I think that's fine for unit testing but do you have a course id so I can try it out on my machine?

Reply:
Sorry, I interpreted that wrong.  8.01SC has Open Learning Library Entries.